File: ShaderUnit/TestScripts/CustomCubeMips.py
# Test custom cubemap mipmap generation.
import utils

#utils.TestSetting('sampler', 'linear', globals())
utils.TestSetting('mip', 1, globals())
filename = 'CustomDownsample_Cube.hlsl'
tex = ri.LoadTexture('Assets/Textures/Cubemap.dds', generateMips = filename)
utils.TestCubemapLevel(ri, tex, mip)

File: SRPTests/TestScripts/CS/CustomCubeMips.cs
// Test custom cubemap mipmap generation.
#load "Utils/utils.cs"

var mip = TestSetting<long>("mip", 1);
var filename = "CustomDownsample_Cube.hlsl";
var tex = ri.LoadTexture("Assets/Textures/Cubemap.dds")
	.WithCustomMips(filename);

TestCubemapLevel(tex, mip);
